star-rating/README.md

```diff
@@ -1,13 +1,68 @@
-# Star Rating Field-Type
+<p style="text-align: center">
+  <a href="https://www.storyblok.com/" rel="noopener">
+ <img width=150px src="https://a.storyblok.com/f/51376/3856x824/fea44d52a9/colored-full.png" alt="Storyblok" style="display:inline-block;margin:20px;"></a>
+</p>
 
-This field-type displays a star rating.
+<h1>
+<img src="assets/icon.svg" height="26px">
+Star Rating Field Type
+</h1>
 
-Name | Description | Author
------------- | ------------- | -------------
-Star Rating | A field-type to display a star rating | [Lisi Linhart](https://github.com/lisilinhart/)
+A field-type to display a star rating.
 
+![screenshot.gif](docs/screenshot.png)
 
-## Amount of stars
+![demo.gif](docs/demo.gif)
 
-The default amount of stars is 5.
-You can change the amount of stars by passing an option with the name `amount` and the value should be the amount of stars you want.
\ No newline at end of file
+## How to use
```

Comment:
This section is more about "setup", and we're missing "Getting Started" section where you run `yarn dev` and run it locally.

Reply:
Let's not think too much about updating everything in the same way. I'd love to get this done quickly, and have you spend your time on the Manifest :)